client/src/charts/canada/canada.js

```diff
@@ -66,7 +69,9 @@ class CanadaGraphBarLegend extends React.Component {
               {`${d.data.year}: ${formatter(d.formattedValue)}`}
             </tspan>
           )}
-          colors={(_d) => get_graph_color(0.5)}
+          colors={(_d) =>
+            color(interpolateBlues(0.5)).copy({ opacity: 0.8 }).toString()
+          }
```

Comment:
The province fill-color take the outputted color and apply a 0.8 opacity. It should probably be documented, but if you want to match the color in legends it's also a good idea to apply the same opacity.

Reply:
Leaving this as a comment on the line's good enough for me :+1: